kraken-app/kraken-app-agent/pom.xml

```diff
@@ -5,12 +5,12 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>kraken-app-agent</artifactId>
     <name>kraken-app-agent</name>
-    <version>1.0.0</version>
+    <version>2.0.0-snapshot.0</version>
```

Comment:
should it be 1.0.0 here ? Maybe need to exclude the bump pr

Reply:
change to 1.1.0-snapshot.0